src/lib/config/types.ts

```diff
@@ -72,6 +72,8 @@ export interface Config {
   slug: string;
   network: string;
   unknown: boolean;
+  inNetworkSelector: boolean;
```

Comment:
How about?
```suggestion
  uiSupported: boolean;
```

Reply:
Makes sense. I've made it `visibleInUI` to be more explicit that enabling this makes it show in the menu's etc.